scipy/integrate/_ivp/radau.py

```diff
@@ -72,6 +72,11 @@ def solve_collocation_system(fun, t, y, h, Z0, scale, tol,
     solve_lu : callable
         Callable which solves a linear system given a LU decomposition. The
         signature is ``solve_lu(LU, b)``.
+    mass_matrix : {None, array_like, sparse_matrix}, optional
```

Comment:
I think the new sparse array is supposed to replace sparse matrix eventually. Doesn't affect the code much, but probably should advertise that sparse array is accepted.